src/modules/capabilities.json

```diff
@@ -0,0 +1,902 @@
+{
```

Comment:
Some of the new changes you made has tabs instead of spaces, can you fix it? the formatting looks off